### PR TITLE
Chore/wait for postgres

### PR DESCRIPTION
**Pull request by stuart-webster-seb**
Port the wait_for_postgres shell script to alpine and set it as the entrypoint of the app container.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,11 +3,12 @@ WORKDIR /app
...
 
-
 COPY lib ./lib
 COPY public ./public
 COPY settings.js ./
 
 RUN npm i
 
+COPY wait_for_postgres.sh /
+ENTRYPOINT [ "/wait_for_postgres.sh" ]
```
Comment (Graborg) [inline Dockerfile]: wait_for_postgres.sh is probably not something that should be included in the production code of this service. I see it more as a tool for easier development. I would suggest moving it back to the Docker compose.

File: docker-compose.yml [hunk 2]
```diff
@@ -19,9 +18,7 @@ services:
       - client_id=
       - client_secret=
       - state_password=
-    # working_dir: /app
-    # entrypoint: ["/wait_for_postgres.sh", "db"]
-    command: tail -f /dev/null
+#   command: ["tail", "-f", "/dev/null"]
```
Comment (Graborg) [inline docker-compose.yml]: I think we can safely remove this line. If it's needed it is possible to bring it back later.

File: wait_for_postgres.sh [hunk 3]
```diff
@@ -1,15 +1,11 @@
...
+#!/bin/ash
 
 set -e
 
-host="$1"
-shift
-cmd="$@"
-
-until </dev/tcp/$host/5432; do
+until nc -z $db_host 5432; do
```
Comment (Graborg) [inline wait_for_postgres.sh]: Nice that this now works with the alpine image!

File: Dockerfile (hunk 1 above)
Comment (stuart-webster-seb) [reply Dockerfile]: Done.

**Pull request merged**